backend/src/controllers/classController.ts

```diff
@@ -1,18 +1,67 @@
 import { Request, Response } from 'express';
 import ClassesModel from '../models/classModel.js';
+import ScheduleModel from '../models/scheduleModel.js';
+import { Class, Schedule } from '../common/interfaces.js'
 
 export default class ClassesController {
 
 	public async getAllClasses(req: Request, res: Response){
 		const classesModel = new ClassesModel();
```

Comment:
Can you move this to top of the class rather than putting it in each function?

We can reduce redundancy since we're going to need it in both functions (and likely future functions)